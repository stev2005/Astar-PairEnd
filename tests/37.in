TTGAGTTAAACGTGGGTAACCCCGTGCCTACGTAGCTGCTGTGCCATAATCCCTAACCCACTTATCCGTTGCGATTAGTAGAGTTGTATCCAAAATCCCCATGGTCCTAAAGCGCAAACTCGTCGTATAGACCTTCACAACAGGGTTGAACCGATATTTACACTTAATCAATCGCTGAGGATTGAGGTGGTTGAGCCGAGTACATTTCTCTACG
15
CCCATTAGCACGCTGAGTCGGAGTATGAAAGCCTGAGGTTGACATAAGTCACGGAGGGGGGAAAAG
GATGGTTGTTGCGACTGATAGCGAAGAAGGACGATCTGTGGGTAGCAACTATCGTCAACTGCACCTGCCCTTTTACAACCTCAATTCATGTGATTCGCGTGGTGCTTCTCGGGGTGCAGCTCTTAGAGTTCCCATGTTTGACCGTAAAAGCTTAGT
TCCGTTTTGCTCACCCGAAGAATTGGGGCG
ATGGGCCGAGCCGGTTCTTTTTGGCCCAGTCGGTACCCTCTAGCGCATATTAGCGAGTAAGCGCAGGCTCGGCCTTGTTTGGTCTCCGCCGTG
AGTGATTGCATCAGATC
CAAACCGGGCG
GGGCCGAACGCATCGTATGCTGCCTATCCCCTAAACGCCTTGAGTGCTGAACGGGGGGTTTCGTCGATTGTGATAACGTTGACAGTGACGTCTCAATCTTTGCTCGTTATGGTTGCTCCATACGCGGA
CTGACCCTCATATCTGTACACGTCTCGCTGACGGCTAGGCTCCGGAACAGGCACGTTACGGCT
AATATCCGGGAAGAGGGATGCCGACTTTAGAATTAGCCATTAACAGTTGGCTTTAAGTATCATATAGACGACGAGTGGGAATTATTACGAAAAAAAATACCAGAACTGTCGTAGTTCACAACACCACCTCTCGCCGGACCCTCCCACTAGTCTTGCTTGTACAG
CCCCGTGCATTAAGCGCGGAATTTCGTTGGGTTTACGGGGGCTGTACAGTAGGACTGAGCGAACACGGTCCCGAAGACGGATAGTCCCCAGAATCAAATTCACTACCAGAGGTGAGTCT
CGAGCGGCGCATCCGCGACAATGTCGCAAACCGCTTTGACTCACGTGCTACTTAGCGTCGTGAGATCTCGCCTCGCAAGTAAGAAACGTGCTCCCCAGAGACTTTCCTCTTGTCGTGTGCCTAGACTAAAGACCAAGCAAATGAAATGTCAAAATAGGCGGTGACGATTATA
CAACTGACGCCTAGCGTTCGCCT
CTTGGTCATCCTAGAGTCCTTGCGCGCTTATCTGTGGAGCGTAGCAAAGGTCACAGTCCTCAACGTTATCGCAGTGTAGCGGGGTGAGTCCCCCGAACAAGGCGCCAACTCTCTCAGGGCTAGCCACGCCGTATGCAGCCCGAGCGGTACCTCTAG
CACTGCTGTAGCCATTACAGAAGCCGGAGAAACGTTTGCGGATTAGGAAGTAGCCTTTAGTAGAGCACTCTCCTACCGGCACCTGGGGCGACTGCCTCTGTGTACTGTCAACCGATATCGCCTATAGGGCACTTCCGTAT
ACAGCATACCGTAGGAACGCCTCCCGTAGTAGAACCAAGGCACCGTCTAAAGTCTATGCCCCA
