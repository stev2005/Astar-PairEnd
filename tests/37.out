30
74
13
47
6
3
59
29
80
55
80
9
74
62
30
